libs/dal/src/repositories/change/change.repository.ts

```diff
@@ -1,23 +1,23 @@
 import { ChangeEntityTypeEnum } from '@novu/shared';
 
-import { EnforceEnvOrOrgIds } from '../../types/enforce';
+import { EnforceEnvId } from '../../types/enforce';
 import { BaseRepository } from '../base-repository';
 import { ChangeEntity, ChangeDBModel } from './change.entity';
 import { Change } from './change.schema';
 
-export class ChangeRepository extends BaseRepository<ChangeDBModel, ChangeEntity, EnforceEnvOrOrgIds> {
+export class ChangeRepository extends BaseRepository<ChangeDBModel, ChangeEntity, EnforceEnvId> {
```

Comment:
Decided to remove the queries with OrganizationId where is not needed to reduce the amount of indexes, still not entirely sure regarding this collection.

Reply:
Yeah that why i raised here, i wanted to check first on the tests level and then validate with David :)